packages/manageiq-ansible-venv/Dockerfile

```diff
@@ -14,27 +14,25 @@ ENV RPM_SPEC=manageiq-ansible-venv.spec \
 # CentOS repos: rpm-build and copr-cli dependencies
 # EPEL: copr-cli
 RUN sed -i 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/subscription-manager.conf
-RUN dnf -y install \
+RUN dnf config-manager --setopt=tsflags=nodocs --save && \
```

Comment:
We should probably set ARCH here like we do in the other images rather than expecting a build arg.
https://github.com/ManageIQ/manageiq-rpm_build/blob/8cc957a8dd8d2068335a64c65361156e3e2f72f8/Dockerfile#L7